array-manipulation-1/src/tail.js

```diff
@@ -0,0 +1,15 @@
+/* exported tail */
+/* receive array argument
+iterate through all indices of the array
+push the second to last elements into a new variable
+return that variable */
+
+function tail(array) {
+  var tailArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i > 0) {
```

Comment:
Instead of having a conditional run with every iteration, you should just start the loop one step later.